base/setup/lib/setuplib.c

```diff
@@ -477,7 +465,7 @@ GetSourcePaths(
 
     /* Check whether the resolved \SystemRoot is a prefix of the image file path */
     // FIXME: commented out to allow installation from USB
-    // if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))
+    /**/if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))/**/
```

Comment:
```suggestion
    //if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))
```

Reply:
This is part of one of these temporary test commits, that won't be included in the final PR version for merging.